core/domain/event_services.py

```diff
@@ -93,6 +94,57 @@ def _handle_event(
                 rule_spec_index, classification_categorization, params,
                 session_id, time_spent_in_secs))
 
+        is_feedback_useful = True
+        if classification_categorization == (
+                exp_domain.DEFAULT_OUTCOME_CLASSIFICATION):
+            is_feedback_useful = False
+
+        stats_models.AnswerSubmittedEventLogEntryModel.create(
```

Comment:
why is this (a) not part of record_answer, (b) not hidden under ENABLE_NEW_STATS_FRAMEWORK?

Reply:
@seanlip Didn't see why we need it inside record_answer. Record answer stores the submitted answer for visualisation purposes, while we are only recording the answer submitted event.
But, yes it should be under ENABLE_NEW_STATS_FRAMEWORK.

Reply:
Ah ok. You're right on both counts. Thanks!